lib/src/client.dart

```diff
@@ -125,6 +125,35 @@ class MeiliSearchClient {
     return Task.fromMap(response.data!);
   }
 
+  /// Get the status of all experimental features that can be toggled at runtime
+  @RequiredMeiliServerVersion('1.3.0')
+  Future<ExperimentalFeatures> getExperimentalFeatures() async {
```

Comment:
We did not provide these kinds of methods to the other SDKs. I understand they are very convenient during development but we believe they are not useful after that, since you only enable/disable once :)

Reply:
I can mark them as internal as a warning to users to not use this

Reply:
I think that's ok to keep, my concern is because it is more code to maintain and it will be present only in this SDK :)